packages/zone.js/package.json

```
@@ -19,7 +19,7 @@
     "test": "test"
   },
   "devDependencies": {
-    "@types/node": "^10.9.4",
+    "@types/node": "^14.0.5",
```

```suggestion
    "@types/node": "^14.0.27",
```
Upgrade `@types/node` to last version.

---

Yeah, we have to wait for node in `angular` repo update to `14.0.27+` and continue on this PR later.

---

I see. Nevermind!